backend/app/routers/matching.py

```
@@ -1,3 +1,36 @@
+async def approve_chain(
+    chain_id: int,
+    current_user: User = Depends(get_current_user),
+    db: AsyncSession = Depends(get_db),
+):
+    return await matching_service.approve_chain(db, chain_id)
+
+
+@router.patch("/relay-chains/{chain_id}/reject")
+async def reject_chain(
+    chain_id: int,
+    current_user: User = Depends(get_current_user),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)